library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
-    * Other-physiologic-pattern {requireChild}
+    * Rhythmic-activity-pattern <nowiki> {suggestedTag=Rhythmic-activity-attribute, suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern} [Rhythmic activity. (Source: Beniczky ea 2017, Table 14.)]</nowiki>
+    * Slow-alpha-variant-rhythm <nowiki> {suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern} [Characteristic rhythms mostly at 4-5 Hz, recorded most prominently over the posterior regions of the head. Generally alternate, or are intermixed, with alpha rhythm to which they often are harmonically related. Amplitude varies but is frequently close to 50 micro V. Blocked or attenuated by attention, especially visual, and mental effort. Comment: slow alpha variant rhythms should be distinguished from posterior slow waves characteristic of children and adolescents and occasionally seen in young adults. (Source: Beniczky ea 2013, Appendix S6; Beniczky ea 2017, Table 14.)]</nowiki>
+    * Fast-alpha-variant-rhythm <nowiki> {suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern} [Characteristic rhythm at 14-20 Hz, detected most prominently over the posterior regions of the head. May alternate or be intermixed with alpha rhythm. Blocked or attenuated by attention, especially visual, and mental effort. (Source: Beniczky ea 2013, Appendix S6; Beniczky ea 2017, Table 14.)]</nowiki>
+    * Lambda-wave <nowiki> {suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern} [Diphasic sharp transient occurring over occipital regions of the head of waking subjects during visual exploration. The main component is positive relative to other areas. Time-locked to saccadic eye movement. Amplitude varies but is generally below 50 micro V. (Source: Beniczky ea 2013; Appendix S6, Beniczky ea 2017, Table 14.)]</nowiki>
+    * Posterior-slow-waves-youth <nowiki> {suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern}[Waves in the delta and theta range, of variable form, lasting 0.35 to 0.5 s or longer without any consistent periodicity, found in the range of 6-12 years (occasionally seen in young adults). Alpha waves are almost always intermingled or superimposed. Reactive similar to alpha activity. (Source: Beniczky ea 2013; Appendix S6, Beniczky ea 2017, Table 14.)]</nowiki>
```

Comment:
micro V  -->  microV 

(to be consistent with descriptions elsewhere)

Reply:
fixed